### PR TITLE
Week 1 - Functions

### PR DESCRIPTION
**Pull request by sarahhae**
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
8.5/10

Was there anything that you struggled with?;
Yes, I struggled with the 'return' function and the equations. I tried to work on the strings homework, but I was unable to find the correct way to start and continue. 

Is there anything that you'd like some further information on?;
I would like more help on when and where to use return, output, if and else if. 

Roughly how long did it take?
3hrs~ for calculations, 2hr~ strings (mainly researching)

### CODE REVIEW

File: Sarah Hong/Homework/Week-01/Wednesday/Arrays & Functions /index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<head>
+  </head>
+  <title>Javascript Excercises</title>
+  <script src="js/homework_02.js"></script>
+  <meta charset="utf-8">
+
+  <body>
+    <p>Please check the Javascript console for program output.</p>
+    </body>
+</html>
```
Comment (CamillaCdC) [inline Sarah Hong/Homework/Week-01/Wednesday/Arrays & Functions /index.html:11]: This is almost right (I'm assuming it worked, but when we get to writing more HTML you'll have issues). It should look like: 

```
<html>
  <head>
    <title>Javascript Excercises</title>
    <script src="js/homework_02.js"></script>
    <meta charset="utf-8">
  </head>

  <body>
    <p>Please check the Javascript console for program output.</p>
  </body>
</html>
```

File: Sarah Hong/Homework/Week-01/Wednesday/Arrays & Functions /js/homework_02.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  } else if ( character === 'i') {
+    return true;
+  } else if ( character === 'o' ) {
+    return true;
+  } else if ( character === 'u' ) {
+    return true;
+  } else {
+    return false;
+  }
+}
```
Comment (CamillaCdC) [inline Sarah Hong/Homework/Week-01/Wednesday/Arrays & Functions /js/homework_02.js:57]: This looks good. If you wanted to make this shorter you could do something like: 
const vowels = function ( character ) {
  if ( character === 'a' ||  character === 'e' || character === 'i' || character === 'o' || character === 'u' ) {
    return true;
  } else {
    return false;
  }
}

OR as the instructions suggest create an array and loop through that. (I'll leave that one to you if you want to give that one another go)

**Comment by CamillaCdC** [comment]
Your indentation and spacing is amazing! Thank you. Makes it so much easier to read. 
Let me know if the return, output, if and else if stuff is still confusing you, and we can find some time to do a couple of exercises together.

**Comment by sarahhae** [comment]
**How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible**

6  / 10

**Was there anything that you struggled with?**

I struggle with what I need to look up to research specific information. I think I'm getting better, but still takes the majority of my time trying to find one specific thing - this is probably because of my lack of terminology still. 

**Is there anything that you'd like some further information on?**



**Roughly how long did it take?**

4 hours - part 1,  ~30min part 2,  question 1. I wasn't able to tackle the guessing game yet!

File: Sarah Hong/Homework/Week-01/Wednesday/Arrays & Functions /js/homework_02.js (hunk 2 above)
Comment (sarahhae) [reply Sarah Hong/Homework/Week-01/Wednesday/Arrays & Functions /js/homework_02.js:57]: Thanks Camilla! 
I've tried the shortened version you mentioned above before, but everything console.logged to all 'true' or all 'false'. I'll try again - I must have added something different that made it not work. 
And yes, ill have a crack on the array and loop solution and let you know how I go :)

**Pull request merged**